nmosquery/etcd_watch.py

```diff
@@ -129,7 +129,7 @@ def _wait_event(self, since):
                 next_index_param = "&waitIndex={}".format(current_index + 1)
                 req = requests.get(self._long_poll_url + next_index_param, proxies={'http': ''}, timeout=20)
 
-            except socket.timeout:
+            except (socket.timeout, requests.ReadTimeout):
```

Comment:
Should it be requests.exceptions.ReadTimeout?

Reply:
It could be, but it's also available as `requests.ReadTimeout` (to avoid having to also do an additional import at)